library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -499,7 +499,7 @@ public void testRecoveryOfBlockMetadataIterator() throws InterruptedException
     testMeta.fileSplitterInput.getScanner().setScanIntervalMillis(500);
 
 
-    testMeta.fileSplitterInput.setup(testMeta.context);
```

Comment:
@chandnisingh
The tests are not supposed to call setup() multiple times for the same operator instance. According to the Java doc of the `Component.setup()` callback to setup() "is made exactly once during the operator lifetime." If a test relies on setup() to clean or reinitialize an operator state, such test should be modified.

It will be good to fix all operators that are frequently used. Please file new JIRA for such operators.